src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3212,6 +3221,15 @@
   <data name="Overflow_Decimal" xml:space="preserve">
     <value>Value was either too large or too small for a Decimal.</value>
   </data>
+  <data name="Overflow_Decimal32" xml:space="preserve">
+    <value>Value was either too large or too small for a Decimal32.</value>
+  </data>
+  <data name="Overflow_Decimal64" xml:space="preserve">
+    <value>Value was either too large or too small for a Decimal64.</value>
+  </data>
+  <data name="Overflow_Decimal128" xml:space="preserve">
+    <value>Value was either too large or too small for a Decimal128.</value>
+  </data>
```

Comment:
I'm sure I'll get to the code that uses this in a bit, but I wouldn't expect these to be necessary.

`Decimal32/64/128` are like `half/float/double`, not like `decimal` and so should "overflow" to infinity and never throw due to a value being "unrepresentable"